lib/money/core_extensions.rb

```diff
@@ -2,8 +2,8 @@
 #   100.to_money => #<Money @cents=10000>
 #   100.37.to_money => #<Money @cents=10037>
 class Numeric
-  def to_money
-    Money.new(self)
+  def to_money(currency = Money.default_currency)
```

Comment:
This doesn't seem consistent with `Money.new`'s currency defaulting to `nil`. Either this should be match `Money.new` or make it match this format. Alternatively you could splat the args and pass any number of args to `Money.new`